app/src/main/assets/buildings.json

```diff
@@ -1,7 +1,6 @@
 {
     "SGW_buildings": [{
             "name": "Henry F. Hall Building",
-            "initial": "H",
```

Comment:
I forgot to mention in the commit message:
I removed these initials in the json as they are not needed. The initials are actually various icons from this path
...\SOEN390-CampusGuideMap\app\src\main\res\mipmap\ic_building_h.png

Reply:
@JeremiahTee could you please remove them? It was added via @Loujain-Al-Nakeeb merging the master onto this branch to fix merge conflicts 😄

Reply:
I removed them in 33a8012 😃